cpp/src/cider-velox/src/CiderPipelineOperator.cpp

```diff
@@ -43,10 +43,8 @@ void CiderPipelineOperator::addInput(RowVectorPtr input) {
   this->input_ = std::move(input);
   ArrowArray* inputArrowArray = CiderBatchUtils::allocateArrowArray();
   exportToArrow(input_, *inputArrowArray);
-  ArrowSchema* inputArrowSchema = CiderBatchUtils::allocateArrowSchema();
-  exportToArrow(input_, *inputArrowSchema);
 
-  batchProcessor_->processNextBatch(inputArrowArray, inputArrowSchema);
```

Comment:
talked with xieqi before, we may need the schema to do validate staff.

Reply:
since your benchmark doesn't use this file, maybe a comment is better, change the public API need a discussion through we don't use the schema in the cider now.

Reply:
since your benchmark doesn't use this file a comment is better through cider doesn't use schema now.

Reply:
since `try to remove schema import which boost 7%`, validating the schema for every batch seems to have no benefit, and velox also doesn't validate every row vector.

Reply:
seems the encoded string support need the schema info, can it be exported once and kept/passed to cider?

Reply:
@jikunshang seems does not check schema？ just check `dictionary` in Array. 
If we need schema once, should split it from `processNextBatch`.

Reply:
yes, you are right. I remember we will not pass `ArrowSchema` to runtime context.

Reply:
Then it makes sense to hack the schema.